packages/webapp/pages/account/profile.tsx

```diff
@@ -94,7 +94,7 @@ const AccountProfilePage = (): ReactElement => {
       }),
     {
       onSuccess: async (res) => {
-        await updateUser({ ...user, cover: res.user.cover });
+        await updateUser({ ...user, cover: res.user.cover } as LoggedUser);
```

Comment:
it does not infer?  ah probably because `user` can be `undefined`

Reply:
It's the difference between `LoggedUser` and `AnnonymousUser` one has mandatory cover, other doesn't basically.